horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/UtilTest.java

```diff
@@ -118,6 +121,41 @@ public void close() throws IOException {
         }
     }
 
+    @Test
+    public void findJsonPath() throws UnsupportedEncodingException {
+        ObjectNode root = JsonNodeFactory.instance.objectNode();
+        ObjectNode metrics = root.putObject("metrics");
+        ArrayNode jobSummary = metrics.putArray("jobSummary");
+        ArrayNode values = JsonNodeFactory.instance.arrayNode();
+        ObjectNode valuesWrapper = JsonNodeFactory.instance.objectNode();
+        valuesWrapper.put("values", values);
+        jobSummary.add(valuesWrapper);
+
+        ObjectNode gcSummary = JsonNodeFactory.instance.objectNode();
+        String gcEnd = "2024-09-05T18:16:36.803661713Z";
+        gcSummary.put("endTimestamp", gcEnd);
+        String gcStart = "2024-09-05T18:14:42.798970958Z";
+        gcSummary.put("timestamp", gcStart);
+        values.add(gcSummary);
+        ObjectNode gcJobConfig = gcSummary.putObject("jobConfig");
+        gcJobConfig.put("name", "garbage-collection");
+
+        ObjectNode cdv2Summary = JsonNodeFactory.instance.objectNode();
+        String cdv2End = "2024-09-05T18:14:42.797624736Z";
+        cdv2Summary.put("endTimestamp", cdv2End);
+        String cdv2Start = "2024-09-05T17:49:02.915410546Z";
+        cdv2Summary.put("timestamp", cdv2Start);
+        values.add(cdv2Summary);
+        ObjectNode cdv2JobConfig = cdv2Summary.putObject("jobConfig");
+        cdv2JobConfig.put("name", "cluster-density-v2");
+
+        System.out.println(root);
```

Comment:
Please can you either remove this println, or replace it with a `Log.trace(root);` statement, so that we can control logging while running tests, thanks

Reply:
Removed